aten/src/ATen/Dimname.h

```diff
@@ -20,6 +20,10 @@ struct CAFFE2_API Dimname {
 
   bool can_refer_to(const Dimname& other) const;
 
+  bool is_normal() const { return type_ == NameType::NORMAL; }
```

Comment:
I don't have a better suggestion right now but we may eventually want to come up with a more explicit name for 'normal'. I feel like the word 'normal' doesn't convey any information without knowing that the other options are wildcard and tagged.

Reply:
I'd like to rename it to "simple" sometime in the future